jetson/nav/README.md

```diff
@@ -0,0 +1,151 @@
+# <img src="../../simulators/nav/src/static/mrover.png" alt="MRover Logo" width="30"/> MRover Navigation Codebase
```

Comment:
Seems to be some git issues with this. Joe is there any way you can remove this? Idk if a rebase from main would help